training/distributed_training/pytorch/model_parallel/gpt2/smp-fine-tune-gpt-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        "fine_tune": 1,
</pre><p>Add a comment above this line</p><pre class="ql-syntax" spellcheck="false"># parameters to enable the fine-tuning mode and specify the name of the model to load
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>